app/assets/styles/map.scss

```diff
@@ -0,0 +1,29 @@
+#map {
+  height: 100%;
+  width: 100%;
+}
+
+#list {
+  height: 95%;
```

Comment:
Hm that is pretty hacky. Usually it'd be good to define an outer div encapsulating both elements but since I'm guessing you're talking about the header, which I'm assuming is the nav, we can't do that here. I think we can use this for now and I'll look more into how to structure the nav - there should be some set up to have nav not interfere with rest of page.